cmd/login.go

```diff
@@ -260,10 +261,25 @@ func getCertFromServer(ctx *cli.Context, cf *config.ServerConfig) (*cliclient.Ma
 
 	req.SetBasicAuth(cf.AccessKey, cf.SecretKey)
 
+	var proxy func(*http.Request) (*url.URL, error)
+	if cf.ProxyURL != "" {
+		proxyURL, err := url.Parse(cf.ProxyURL)
+		if err != nil {
+			return nil, fmt.Errorf("invalid proxy address %s: %w", cf.ProxyURL, err)
+		}
+		proxy = http.ProxyURL(proxyURL)
+	} else {
+		proxy = http.ProxyFromEnvironment
+	}
```

Comment:
This can potentially be extracted into a helper to share between login and ssh commands.

Reply:
Done.